### ISSUE TITLE
ZeroDivisionError: float division by zero

### ISSUE THREAD

**Issue by theevilbit** [fireeye/flare-floss#8]
Used the compiled version on Ubuntu, and get the following for the sample with MD5: 575372f2ad8021b9d12da1213bc1d1bf

```
./floss Order\ 150216.exe
WARNING:__main__.StringDecoderConfig:load_plugins: failed to load extra plugins: The 'floss' distribution was not found and is required by the application
Traceback (most recent call last):
  File "<string>", line 428, in <module>
  File "<string>", line 414, in main
  File "<string>", line 51, in identify_decoding_functions
  File "<string>", line 341, in run_plugins
  File "floss/plugins/function_meta_data_plugin.py", line 40, in score
ZeroDivisionError: float division by zero
```

**Comment by williballenthin** [comment]
Good catch. We'll get this fixed today. Thanks!

**Comment by mr-tz** [comment]
fixed via 524fa2f9fd0d1ff3870323e655dc4d2834623fac
